docs/recipes/developing-and-testing/debug-safe-app.md

```diff
@@ -85,23 +85,54 @@ Click the "Add Configuration..." button and choose **Launch .NET Core Console Ap
 The only change required is to point it at the Server application, by replacing the `program` line with this:
 
 ```json
-"program": "${workspaceFolder}/src/Server/bin/Debug/netcoreapp3.1/Server.dll",
+"program": "${workspaceFolder}/src/Server/bin/Debug/net5.0/Server.dll",
 ```
 
 #### 3. Configure a build task
 * From the Command Palette, choose `Configure Task`.
 * Select `Create tasks.json file from template`. This will show you a list of pre-configured templates.
 * Select `.NET Core`.
+* Update the build directory using `"options": {"cwd": "src/Server},` as shown below:
```

Comment:
Missing close quote

Reply:
Ah, well spotted. @isaacabraham I just updated it on the 'alfonso pr comments' branch I have open.